### DISCUSSION TITLE
System Environment Variables docs seem to be wrong?

### DISCUSSION THREAD

**Discussion by chrstphrknwtn** [vercel/vercel#4375] [General] [answered]
Inspecting the output of `process.env` during the build step reveals that the new `VERCEL_*` env vars are actually still the `NOW_*` env vars.

Am I missing some setting where I get the old ones vs the new ones?

**Comment by lucleray** [comment, marked as answer]
Hi @chrstphrknwtn,

The environment variables you are seeing are legacy environment variables that were automatically added to all deployments before we released the new [System Environment Variables](https://vercel.com/docs/v2/build-step#system-environment-variables).

For backwards-compatibility reasons, we are still adding these legacy environment variables to projects that meet both of these conditions simultaneously:
- the project is not using the new [System Environment Variables](https://vercel.com/docs/v2/build-step#system-environment-variables)
- the project was created [before April 30, 2020 (UTC)](https://vercel.com/blog/changelog-april-2020#projects)

To use the new `VERCEL_*` [System Environment Variables](https://vercel.com/docs/v2/build-step#system-environment-variables), you need to explicitly add it to your project. For example, add `VERCEL_URL` to your project. You can then consume it with `process.env.VERCEL_URL` during Build Step.
